components/Impact.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @next/next/no-img-element */
+const HEADER_CONTAINER = "bg-[#7055FD] text-white font-semibold text-4xl p-10 py-15 hidden md:flex";
+const IMPACT_CONTAINER = "bg-white w-full my-10 hidden md:flex";
+const IMPACT_MOBILE_CONTAINER = "bg-white w-full my-5 md:hidden";
+
+const IMPACT_SVG = "/svgs/impact/impact.svg";
+const IMPACT_MOBILE_SVG = "/svgs/impact/impact_mobile.svg";
+
+const Impact = () => {
+  return (
+    <>
+      <div className={HEADER_CONTAINER}>Over the course of our efforts, we have...</div>

```

Comment:
This seems to be aligned with the logo in the navbar in the figma. Add more padding to the left